src/Evolution/Imex/Protocols/ImplicitSector.hpp

```diff
@@ -93,7 +93,7 @@ namespace imex::protocols {
 /// jacobian:
 ///
 /// \snippet Test_SolveImplicitSector.cpp source
-/// \snippet Test_SolveImplicitSector.cpp jacobian
+/// \snippet Test_SolveImplicitSector.cpp Jacobian
```

Comment:
Mention/link to the new protocols from the docs in this file, and check in this protocol's `test` that the sub-structs conform to the new protocols.